locales_dev/en/translation.json

```diff
@@ -374,6 +377,8 @@
   "confirmRemoveCollaboratorsMessage": "The following user is being removed from this epic, but is already assigned to at least one task. Removing this user will not remove them from any assigned tasks. Are you sure you want to remove this user from the epic?",
   "confirmRemoveCollaboratorsMessage_plural": "The following user is being removed from this epic, but is already assigned to at least one task. Removing this user will not remove them from any assigned tasks. Are you sure you want to remove this user from the epic?",
   "contributeWorkFromEpic": "<0><0>To contribute the work you’ve done in your scratch org,</0> you’ll start by creating a new task.</0><1>Your scratch org will become the Dev Org for the newly created task.</1>",
```

Comment:
Is there a distinction between lowercase `scratch org` and uppercase `Scratch Org`?

Reply:
That's a great question. I've seen both `Scratch Org` and `scratch org` (similarly both `project` and `Project` etc) in the mockups and the live site, and I'm not 100% sure if we have a clear rule. @SondraE or @stacyk do you want to weigh in here?

It'd probably be a good idea to create a new story to track this, where we can develop and document a clearer style guide.

Reply:
I believe we were leaning towards having all of the Objects capitalized... but I think we were converting those as we moved along and didn't always catch all of the terms retroactively. I know Projects, Tasks, Epics, Collaborators were the things we started to capitalize, but maybe we could develop more concrete guidelines and document them somewhere. @SondraE what is your understanding?

Reply:
That makes sense to me. I'll make a new card to more carefully review and update the current usage. 👍 

https://trello.com/c/J04dg5Yi/293-i-want-consistent-capitalization-of-core-objects-terms-eg-project-task-scratch-org

Reply:
Yep. As I was writing text for the self-guided tour, I was capitalizing Project, Epic, Task, Collaborator, and Org to help me remember what the important objects were. Then I realized capitalizing these words might help users grasp their importance in an application that has a lot of terms to learn.